utils/superset/db_name_change.py

```diff
@@ -0,0 +1,100 @@
+"""
+This script will generate the upload json for entered database name
+"""
+import os
+import json
+import argparse
+import logging
+import sys
+
+
+# create logger
+logging.basicConfig(level=logging.DEBUG)
+
+
+# file extension validation
+def validate_extension(file_name):
+    """
+    This will ensure that the input or output file will have a .json extension
+    """
+    _, ext = os.path.splitext(file_name)
+    if ext.lower() != ('.json') or len(ext) == 0:
+        raise argparse.ArgumentTypeError('File must have JSON extension')
+    return True
+
+def file_handle_error(function):
+    """
+    This decorator is to handle the exceptions while read, write to the file
+    """
+    def exception_handle(*arguments):
+        try:
+            return function(*arguments)
+        except FileNotFoundError:
+            logging.error("File '%s' not found. Aborting", arguments[0])
+            sys.exit(1)
+        except IOError as exception:
+            logging.error("I/O error %s: %s",exception.errno, exception.strerror )
+            sys.exit(1)
+        except Exception as err: # pylint: disable=broad-except
+            logging.error("Unexpected error while writing to the file: %s", err)
+            sys.exit(1)
+    return exception_handle
+
+@file_handle_error
+def read_input_file(input_file,db_name):
+    """
+    This is input file function to read the file content and replace the database name
+    """
+    file_data = {"dashboards":[], "datasources":[]}
+    if validate_extension(input_file):
+        with open(input_file, encoding="utf_8") as file_obj:
```

Comment:
I would look at Pathlib here, it'd make the extraction of the extension nicer, too https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix or even https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match

Reply:
Done. Removed the validation function and now using the PurePath `.suffix` to match the extension. 

PR is ready for your review. Thanks for the suggestions @drsm79!